tests/behat/features/site-fields.feature

```diff
@@ -0,0 +1,52 @@
+@install
+Feature: Site and primary site fields exist.
+
+  Fresh install of tide profile has the site and primary site fields available
```

Comment:
Based on the description this belongs in Tide which I think is also true based on the content of the test :) [Tide does define tide_site as a dependency](https://github.com/dpc-sdp/tide/blob/9768b3728a5cc0b39c6bb290b6c947bf96b3055b/composer.json#L23) so if it extends any functionality that tide_site provides it should be responsible for testing that.

Reply:
@GROwen see Sonny's comment [here](https://github.com/dpc-sdp/tide_site/pull/90#issuecomment-884851842). It can't be tested in `tide` as the builds there don't have the content types installed. It has to be tested in an _instantiation_ of a tide site as I understand it. Thoughts?

Reply:
Further investigation confirms this @GROwen. I built both `tide` and `tide_core` and none of the dev-tools builds are done via the tide profile but directly off composer.json. This means they are composer required but because it's only the `tide` profile that has a _Drupal_ dependency (i.e. in `tide.info.yml`) the `tide_*` modules don't get enabled.

Reply:
Looking at how this is done in Drupal land we would need to change how we define our modules (it seems the requirement for a `composer.json` is gone in Drupal 9 (see [here](https://www.drupal.org/docs/creating-custom-modules/add-a-composerjson-file#s-drupal-9-compatibility)) and it is all generated off `<module>.info.yml`. We would have to majorly rework our CI and testing to support this IMO.

Reply:
@GROwen do you have any more thoughts here? I think this might have to be a future task.

Reply:
To get the content types enabled (and therefore get this working as a test in Tide) could the [`install` property be added to tide.info.yml](https://www.drupal.org/docs/distributions/creating-distributions/how-to-write-a-drupal-installation-profile#s-infoyml-file)?

I may be misunderstanding the blocker to moving this to Tide because I can't understand [this comment from Sonny](https://github.com/dpc-sdp/tide_site/pull/90#issuecomment-884851842)
> At that point, the issue no longer occurs hence the test would be pointless.

Isn't the point of the test to ensure the fields are present on a full install?

Reply:
`tide_test` is installed as a step by the `.ahoy.yml` build installed by `dev-tools` e.g. https://github.com/dpc-sdp/dev-tools/blob/master/.ahoy.yml#L17

The error was never a problem once Drupal was installed (that build step referenced above happens _after_ install) but only for modules that are enabled as part of the profile. The only working profile we have is `tide` and none of our CI at this stage builds using that profile but reference site does. I don't have time to rework CI now so I'm just closing this as too hard.